docs/concepts/registries.md

```diff
@@ -199,15 +199,18 @@ Creating Custom Forge Registries
 
 Custom registries can usually just be a simple map of key to value. This is a common style; however, it forces a hard dependency on the registry being present. It also requires that any data that needs to be synced between sides must be done manually. Custom Forge Registries provide a simple alternative for creating soft dependents along with better management and automatic syncing between sides (unless told otherwise). Since the objects also use a Forge registry, registration becomes standardized in the same way.
 
-Custom Forge Registries are created via `RegistryBuilder` via either `NewRegistryEvent` or the `DeferredRegister`. The class `RegistryBuilder` takes certain parameters (such as the name, the `Class` of its values, and various callbacks for different events happening on the registry). Calling `RegistryBuilder#create` will result in the registry being built, registered to the `RegistryManager`, and returned to the caller for additional processing.
+Custom Forge Registries are created via `RegistryBuilder` via either `NewRegistryEvent` or the `DeferredRegister`. The class `RegistryBuilder` takes certain parameters (such as the name, the `Class` of its values, and various callbacks for different events happening on the registry). `RegistryBuilder#create` is called internally build the registry, register it to the `RegistryManager`, and return the supplier wrapped registry to the caller for additional processing.
+
+!!! warning
+    `RegistryBuilder#create` is package-private and is called through `NewRegistryEvent` or `DeferredRegister`.
```

Comment:
Personally, I wouldn't even mention #create's existence. Modders don't need to know. They just need to know how to get a supplier of their registry. Also, grammar issue, "is called internally build the registry". And the double "via" sounds weird.